ups/rubintv_production.table

```diff
@@ -5,8 +5,9 @@
 setupRequired(atmospec)
 setupRequired(base)
 setupRequired(utils)
-setupRequired(rapid_analysis)
 setupRequired(pex_exceptions)
+setupRequired(summit_utils)
+setupRequired(summit_extras)
```

Comment:
Is `summit_extras` actually required?

Reply:
Hmmm, right now, no, but I thought it wasn't unlikely. But that's fair, I'll remove it for now, can always add it later (I just have forgotten to do so in the past, but that's not a good reason to deliberately get it wrong now!)